Bottom: ae625098e721592d642a82e38d1edabfaf46d7a3
Top:    fe4cb17f4c0c26ecadfff46a5a44815176c4acbd
Author: Ian Kent <raven@themaw.net>
Date:   2013-03-03 09:18:35 +0800

Refresh of autofs-5.0.7-fix-map-read-fail-incorrectly-set-on-master-re-read.patch

---

diff --git a/modules/lookup_file.c b/modules/lookup_file.c
index bca92fd..b6432ff 100644
--- a/modules/lookup_file.c
+++ b/modules/lookup_file.c
@@ -451,12 +451,12 @@ int lookup_read_master(struct master *master, time_t age, void *context)
 				     MODPREFIX
 				     "failed to read included master map %s",
 				     master->name);
-				/*
-				 * Plus map include failures don't cause the map
-				 * read to fail.
-				 */
-				master->read_fail = 0;
 			}
+			/*
+			 * Plus map include failures don't cause the map
+			 * read to fail.
+			 */
+			master->read_fail = 0;
 			master->depth--;
 			master->recurse = 0;
